### PR TITLE
Docker cleanups

### PR DESCRIPTION
**Pull request by mateoclarke**
_most of https://github.com/cityofaustin/techstack/issues/440_

### CODE REVIEW

**Review by luqmaan — COMMENTED** [review]
"node_modules probably shouldn't be a mounted volume to prevent local modification of these."

Isn't it still a mounted volume?

File: docker-compose.yml [hunk 1]
```diff
@@ -13,12 +13,9 @@ services:
     ports:
       - "5000:5000"
     volumes:
-      - node_modules:/app/node_modules
       - .:/app:delegated
+      - /app/node_modules
```
Comment (mateoclarke) [inline docker-compose.yml:17]: @luqmaan The way I understood it via @ifsimicoded is that the way we have it without the `:` followed by a reference directory, it effectively mounts it nowhere. Could be way off on this though.

Comment (ifsimicoded) [reply docker-compose.yml:17]: I was referencing the section [The node_modules Volume Trick](http://jdlm.info/articles/2016/03/06/lessons-building-node-app-docker.html). 
I think what's happening is node_modules is mounted from the image (called an anonymous volume), and not the host, so they're not blown away when the image is run.

Comment (ifsimicoded) [reply docker-compose.yml:17]: But honestly, docker-compose syntax isn't familiar to me, so there's probably a better way that persists these things across docker-compose ups and downs.

**Pull request merged**